packages/eyeglass/src/util/typescriptUtils.ts

```diff
@@ -0,0 +1,7 @@
+export interface Dict<T> {
+  [key: string]: T;
+}
```

Comment:
this is pretty much `Record<string, T>`. Not 100% sure about your planned usage, but my dict types usually are

```ts
interface Dict<T> {
    [k: string]: T | undefined;
}
// a.k.a.
type Dict<T> = { [K in keyof any]?: T }

```

since for static analysis purposes I often can't be sure which keys will have values